source/_posts/2017-05-19-kanban-method-in-a-consulting-company.markdown

```diff
@@ -0,0 +1,100 @@
+which aids decision-making about what, when and how much to produce. Although
+the method (inspired by the Toyota Production System and lean manufacturing
+originated in software development and IT, it may be applied to any professional 
+service whose work outcome is intangible rather than physical.
+
+## Tools
+
+The basic work tools are boards and stickers, but in our company we use visual 
+boards in [Trello] (https://trello.com/), which streamlines our work in kanban. 
+Our arrays are divided into appropriate categories: Backlog, To Do, In Progress,
+Verification, Done and Project Status, where we can discuss project features 
+or problems.
+
+![1](http://imageshack.com/a/img922/6156/YfxSE1.png)
+
+## Cards (Kanbans)
+
+According to the Kanban method we give out a kanban cards to assigned 
+employees. The card is very important in the whole project process. According
+to her, we measure our work and set time for each stage in the design, develop
+and testing solutions in embedded systems projects. Additionally inside the 
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+
+The card life begins in ToDo, sometimes it happens there we have a lot of big
+task and cards with small priority are moved to the Backlog, where they are 
+waiting for some free time to solve them. We are trying to minimize transfering
+the cards from right to left, our goal is to streamline this process.
+
+![kanban-flow](http://imageshack.com/a/img923/2471/u20Zzr.jpg)
+
+## Rules
+
+To make kanban work, each employee must follow these rules:
+
+- [ ] Do not take a lot of tasks,
+- [ ] Keep order,
+- [ ] Discuss about tasks,
+- [ ] Don't race for who will do more - focus on teamwork,
+- [ ] When somebody have problem our task is to help each other.
+
+These rules are only a sample but improve the whole process. We know situations 
+where tasks is going fast to verification, after which it turns out that the 
+task is incomplete and we need turn it back to the left. Good comunication 
+helps, what if some feature is expired and we dont know about it? One sentence 
+in communication can save much of time. 
+
+## Pros and cons
+
+The advantages are:
+
+- Universal - can be used to visualize company overview, project or employee
+individual workflow
+- Creating a WIP (Work In Progress) limit - it prevents overproduction and 
+quickly detects bottlenecks
+- Clarity - one look and you know the whereabouts of the project
+- Stream management - systematic time and effort measurement to optimize 
+processes.
+
+ And cons:
+
+- Need for continuous control
+- Commitment of each employee
+- Incorretly managed will turn into a battlefield where employees struggle, who
+has done more and create a nasty mess on the work cards. 
+
+## Summary
+
+Kanban is designed for companies where software is deployed and delivered in 
```

Comment:
@molszewski9412 not exactly true. Cycles are more like Scrum - Kanban do not define cycle it use workflow and its statisitc measures to make sure we moving with correct pace.